frontend/public/components/_dropdown.scss

```diff
@@ -166,6 +169,51 @@ $color-bookmarker: #DDD;
   }
 }
 
+.co-m-masthead-link {
+
+  display: block;
+  float: none;
+  font-size: 13px;
+  line-height: 23px;
+  overflow-x: hidden;
+  padding: 2px 0 2px $link-padding-left;
+  white-space: nowrap;
+
+  a {
+    background-color: inherit;
+    color: $color-os-nav-title;
+    display: inline-block;
+    text-decoration: none;
+    width: 100%;
+  }
+
+  &.active {
+    padding-left: $link-padding-left - $active-border;
+    background-color: #383F47;
+    border-left: $active-border solid $color-os-nav-border-left-active;
+
+    a {
+      color: #fff;
+    }
+  }
+}
```

Comment:
@rhamilto Would you mind reviewing the styles here?

I suggested a `co-m-masthead-link` class since `co-m-nav-link` seemed wrong., but we might want to share the definition with `co-m-nav-link`

Reply:
This looks like a straight duplication of [`.co-m-nav-link`](https://github.com/openshift/console/blob/master/frontend/public/components/_nav.scss#L6-L30)?  I'd be inclined update _nav.scss with the shared stuff.

```
.co-m-masthead-link,
.co-m-nav-link {
  display: block;
  float: none;
  font-size: 13px;
  line-height: 23px;
  overflow-x: hidden;
  padding: 2px 0 2px $link-padding-left;
  white-space: nowrap;

  a {
    background-color: inherit;
    color: $color-os-nav-title;
    display: inline-block;
    text-decoration: none;
    width: 100%;
  }

  &.active {
    padding-left: $link-padding-left - $active-border;
    background-color: #383F47;
    border-left: $active-border solid $color-os-nav-border-left-active;
    a {
      color: #fff;
    }
  }
}

.co-m-nav-link {
  &:hover {
   background-color: $color-os-nav-background-active;
  }

  &:focus {
   background-color: transparent;
  }
}
```